clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -77,7 +77,7 @@
 
   <xs:simpleType name="connectionUrl">
     <xs:restriction base="xs:anyURI">
-      <xs:pattern value="terracotta://([^\]\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?(/[^\?#]*)?(\?[^#]*)?(#.*)?"/>
+      <xs:pattern value="(terracotta|passthrough)://([^\]\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?(/[^\?#]*)?(\?[^#]*)?(#.*)?"/>
```

Comment:
As long as we have a way to override the scheme, both in XML and programmatic API, OR as long as Ehcache provides a "testing feature" (like a module we could use to test Ehcache, that will contain basically the specific connection service implementation), so that eternal service implementors can use it, then it's fine.